tests/test_decision_maker/test_messages/test_transaction.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains tests for transaction."""
+import pytest
+
+from aea.decision_maker.messages.transaction import TransactionMessage
+
+
+class TestTransaction:
+    """Test the transaction module."""
+
+    def test_message_consistency(self):
+        """Test the message consistency of a transaction message."""
```

Comment:
So message consistency failure is tested here. Please update doc string accordingly.